File: Python/Excel Sheet Column Number.py
class Solution(object):
    def titleToNumber(self, s):
        """
        :type s: str
        :rtype: int
        """
        return sum(map(lambda a: (a[1] - 64) * pow(26, a[0]), list(enumerate(map(ord, s)[::-1]))))

File: cpp/Excel Sheet Column Number.cpp
class Solution {
public:
    int titleToNumber(string s) {
        int  sum = 0;
        int power = 0;
        reverse(s.begin(), s.end());
        for (char ch : s) {
            sum += (ch - 'A' + 1) * pow(26, power++);
        }
        return sum;
    }
};
